lab_3_ann_retriever/main.py

```diff
@@ -50,6 +53,14 @@ def calculate_distance(query_vector: Vector, document_vector: Vector) -> float |
     In case of corrupt input arguments, None is returned.
     """
 
+    if not isinstance(query_vector, list) or not isinstance(document_vector, list):
```

Comment:
what if vector will be incorrect? for example: `[-1, -100, 0, 5]`

Reply:
Done